tests/foreman/sys/test_pulp3_filesystem.py

```diff
@@ -29,15 +28,14 @@ def test_selinux_status(target_sat):
 
     :customerscenario: true
 
-    :BZ: 2131031
+    :BZ: 2263294
```

Comment:
```suggestion
    :BZ: 2131031, 2263294
```

Reply:
The remove BZ is migrated and also the bug is unrelated the test case

Reply:
It looks related to selinux only, and this is ancient test, so we could either keep it but its upto you

Reply:
I would still say that the bug is unrelated to the test so would like to remove the BZ. It would make confusion in future to relate the bug with the test case.